dbstructure.json

```
@@ -43,7 +43,8 @@
                 "metadata": {
                     "location": "123.456, 123.456",
                     "start_time": "",
-                    "end_time": ""
+                    "end_time": "",
+                    "flagged_cnt": 0
```

```suggestion
                    "flag_count": 0
```
i think avoiding abbreviations where possible is a good practice

---

@aidan-hubley I agree with Alex on the abbreviation front; I'll approve this PR once his change is merged (I didn't know this was a feature in Github so we never made a policy for it, but I'll assume we're doing the same as we would for merging PRs, meaning in this case, Aidan needs to make the change)